app/src/main/java/com/taylorgirard/comicconvo/activities/MainActivity.java

```diff
@@ -31,7 +32,7 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_main);
 
         final Fragment[] fragment = new Fragment[1];
```

Comment:
why is this a list?

Reply:
It would not let me edit the fragment within inner classes unless it was "effectively final" and this was the way android studio's quick fix recommended I do so